### DISCUSSION TITLE
new to electrs

### DISCUSSION THREAD

**Discussion by coolanthru** [romanz/electrs#1093] [General]
Hi, 

I installed and ran electrs for the first time last night on linux mint 21. I have some questions to ask after spending many hours researching, learning, etc. 

1. This is kinda vague since I forgot to copy and paste the message, but it was a warning message at the very end of the compiling process right before the final line stating 'finished' with total process time. The warning message was a ResultExt type that mentioned 2 'unused imports' from line 2 of a code file. Is this a concern?

2. After electrs finished indexing and compaction, a bunch of WARN lines followed with electrs::mempool  missing mempool entry  It seems that electrs is caught up with core and indexing as blocks appear, and ocasionally the same mempool warning shows up. What can be causing this?

3. What is the best way to connect to electrs/bitcoin core machine via seperate machine on the home LAN?

Thanks!

**Comment by romanz** [comment]
Thanks for reporting those issues!

1. it shouldn't be a significant issue - but please report it [here](https://github.com/romanz/electrs/issues/new) :)
2. most probably it is caused by mempool transactions being replaced - this WARNING should be a DEBUG-level log...
3. if it's a trusted network, you can just use a plain TCP connection.
